homework-submission/week2/my-app/src/components/ToDoItems.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react"
+import Post from "../Post"
+
+import ToDos from "../Todos"
+
+class TodoItem extends Component {
+   constructor(props) {
+      super(props)
+      this.state = {
+         Todos: ToDos
+      }
```

Comment:
why do you put your data ( Todos ) here, it should be in the App file and pass whatever you need to here as props